packages/shared/src/components/modals/FeedFiltersIntroModal.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from './StyledModal';
+import { ResponsiveModal } from './ResponsiveModal';
+import UserIcon from '../../../icons/user.svg';
+import { Button } from '../buttons/Button';
+import FeedFiltersIntroModalTags from './FeedFiltersIntroModalTags';
+import { FeedFiltersIntroModalTagsContainer } from '../utilities';
+
+type FeedFiltersProps = {
+  actionToOpenFeedFilters: () => unknown;
+  feedFilterModalType: string;
+};
+
+type ModalFooterProps = {
+  feedFilterOnboardingModalType: string;
+  onOpenFeedFilterModal: (event: React.MouseEvent<Element, MouseEvent>) => void;
+} & Pick<ModalProps, 'onRequestClose'>;
+
+type FeedFiltersIntroModalProps = FeedFiltersProps &
+  ModalFooterProps &
+  ModalProps;
+
+const footerClass = {
+  test1: 'justify-center',
+  test2: 'justify-between',
+};
+
+const IntroModalFooter = ({
+  feedFilterOnboardingModalType,
+  onRequestClose,
+  onOpenFeedFilterModal,
+}: ModalFooterProps) => {
+  return (
+    <footer
+      className={classNames(
+        footerClass[feedFilterOnboardingModalType],
+        'flex fixed responsiveModalBreakpoint:sticky bottom-0 py-3 border-t border-theme-divider-tertiary bg-theme-bg-tertiary',
+      )}
+    >
+      {feedFilterOnboardingModalType === 'test2' && (
+        <Button
+          className="ml-4 w-20 text-theme-label-tertiary btn-default"
+          onClick={onRequestClose}
+        >
+          Skip
+        </Button>
+      )}
+
+      <Button
+        className="mr-4 w-40 btn-primary-cabbage"
+        onClick={onOpenFeedFilterModal}
+      >
+        {feedFilterOnboardingModalType === 'test1'
+          ? 'Create my feed'
+          : 'Continue'}
+      </Button>
+    </footer>
+  );
+};
+
+export default function FeedFiltersIntroModal({
+  className,
+  onRequestClose,
+  feedFilterOnboardingModalType,
+  actionToOpenFeedFilters,
+  feedFilterModalType,
+  onOpenFeedFilterModal,
+  ...modalProps
+}: FeedFiltersIntroModalProps): ReactElement {
+  const tagsRows = [
+    ['', 'docker', '', 'kubernetes', ''],
+    ['', '', 'architecture', '', ''],
+    ['', '', '', 'devops', ''],
+    ['', 'cloud', '', '', ''],
+  ];
+
+  return (
+    <ResponsiveModal className={classNames(className)} {...modalProps}>
```

Comment:
I think the className is obsolete here as we don't refer to it on the parent.

```suggestion
    <ResponsiveModal {...modalProps}>
```